tests/markets/test_omen.py

```diff
@@ -186,3 +187,17 @@ def test_resolved_omen_bets(a_bet_from_address: str) -> None:
     # Verify that all bets convert to generic resolved bets.
     for bet in resolved_bets:
         bet.to_generic_resolved_bet()
+
+
+# @pytest.mark.skipif(not RUN_PAID_TESTS, reason="This test costs money to run.")
```

Comment:
I guess this should be uncommented until we have testing chain in place?

(you don't have to comment these while testing it locally, just run tests as `RUN_PAID_TESTS=1 pytest tests/`)

```suggestion
@pytest.mark.skipif(not RUN_PAID_TESTS, reason="This test costs money to run.")
```

Reply:
Nevermind, just saw you already did this in the next PR <3